runtime/doc/lua.txt

```diff
@@ -1728,4 +1728,16 @@ select({items}, {opts}, {on_choice})                         *vim.ui.select()*
                                  1-based index of `item` within `item` . `nil`
                                  if the user aborted the dialog.
 
+==============================================================================
+Lua module: log                                                      *lua-log*
```

Comment:
This is just a table, not a module.

Reply:
where does it go then?

Reply:
Hmm, I guess it should actually be in the `lua-builtin` section, which will not be overwritten.